Makefile

```diff
@@ -51,18 +51,19 @@ endef
 pre-config: pre-init ## Initialise env files, and create some required folders, files and softlinks
 	@echo "You can now edit your config for flavour '$(FLAVOUR)' in config/$(MIX_ENV)/secrets.env, config/$(MIX_ENV)/public.env and ./config/ more generally."
 
+CONFIG_PATH=$(FLAVOUR_PATH)/config
 pre-init:
 	@echo "Setting flavour to $(FLAVOUR_PATH)"
-	@ln -sfn $(FLAVOUR_PATH)/config ./config
+	@ln -sf $(FLAVOUR_PATH)/config ./config
```

Comment:
could you explain why the `n` option was removed?

Reply:
The `n` option doesn't appear to serve any definitive purpose in this context, so I didn't see the need for it. It shouldn't matter if the link is treated like a file or not, the commands worked just as well for me on both operating systems without that flag

Reply:
Docs for `-h` (alias of `-n`): If the link_name or link_dirname is a symbolic link, do not follow it.  This is most useful with the -f option, to replace a symlink which may point to a directory.
When switching flavours, this means that the link is recreated pointing to the new source dir (flavours/x/config).

Reply:
Alright fair enough, thank you for explaining. I'll be sure to switch it back before making my next PR